turbo.json

```diff
@@ -10,20 +10,12 @@
     "clean": {
       "cache": true
     },
-    "storybook-build": {
-      "outputs": [
-        "apps/pie-storybook/build/**"
-      ],
-      "dependsOn": [
-        "^build"
-      ]
-    },
     "storybook-serve": {
       "dependsOn": [
-        "storybook-build"
+        "pie-storybook#build"
       ]
     },
-    "storybook-dev": {
```

Comment:
The new repo command will be `yarn dev --filter=<`package.json `name`>.

e.g - `yarn dev --filter=pie-storybook`
`yarn dev --filter=pie-microsite`

Running `yarn dev` without `--filter` will start dev servers for all packages with a `dev` command.

I'm not sure what your opinion of this is personally, but I think having a standardised `dev` command is a bit easier to remember, even if it means we have to apply `--filter` on the end.

This is what turborepo recommends rather than `cd`ing into directories. This is because any dependencies are handled under-the-hood by Turborepo 👍🏼  - https://turbo.build/repo/docs/core-concepts/monorepos/filtering#filter-by-workspace-name

Reply:
Also, we could write a wiki on how to add an alias to your cmd line, so you can customise whatever you like to run the main repo/docs commands

Reply:
> This is because any dependencies are handled under-the-hood by Turborepo

Fine by me, but what does that mean exactly?

Reply:
> The new repo command will be yarn dev --filter=<package.json name>.

In this case, I think the following scripts should be removed from the root `package.json`: `storybook-build`, `storybook-dev`, and `storybook-serve`

Reply:
Essentially Turborepo is only utilised when running commands from the root, meaning If you were to change directory into `apps/pie-storybook` and run `yarn dev` (or any command) it bypasses Turborepo, meaning that any dependencies (such as pie-button) won't have their build tasks executed 👍🏼.

whilst `storybook-dev` was a nice targeted command to only spin up Storybook, I think in terms of scalability, it's better to have the standard `dev` command across all packages, and just get in the habit of using `--filter`.

Hope that justification makes sense!

Reply:
As Fernando mentioned – does that mean we can get rid of the references to `storybook-x` type scripts in the root `package.json` now then?

Reply:
Ah yes, will remove those!